modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/model/XSKDataStructureModel.java

```diff
@@ -53,7 +53,7 @@
   private transient String rawContent;
 
   @Transient
-  private transient XSKHanaVersion hanaVersion;
+  private transient XSKDBContent dbContentVersion;
```

Comment:
Should it be dbContent instead? If version is no longer applicable?

Reply:
I agree, if the version is no longer  applicable

Reply:
How about dbContentType

Reply:
In my opinion type is better solution, because if it only dbContent, I would expect to get the dbContent, which is not the case and will be tricky.